modulus/models/module.py

```diff
@@ -178,7 +179,7 @@ def save(self, file_name: Union[str, None] = None, verbose: bool = False) -> Non
             torch.save(self.state_dict(), local_path / "model.pt")
 
             with open(local_path / "args.json", "w") as f:
-                json.dump(self._args, f)
+                json.dump(self._args, f, cls=OmegaConfEncoder)
```

Comment:
This is in the other PR, so why is it also here?

Reply:
This was for testing but I forgot to remove it. It is removed now